internal/document/editor/cell.go

```diff
@@ -144,6 +145,11 @@ func toCellsRec(
 
 		case *document.MarkdownBlock:
 			value := block.Value()
+			astNode := block.Unwrap()
+
+			metadata := &map[string]string{}
```

Comment:
```suggestion
			metadata := map[string]string{}
```

`map[K]V{}` is already a pointer (the same for `make(map[K]V)`).

Reply:
It was fixed, Thank you @adambabik :+1: